content/en/users/cloud-compute/openstack/_index.md

```diff
@@ -29,14 +29,39 @@ of this guide will focus on CLI/API access.
 
 ## Installation
 
-TBC
+The OpenStack client is a command-line client for OpenStack that brings the
+command set for Compute, Identity, Image, Object Storage and Block Storage
+APIs together in a single shell with a uniform command structure.
+
+> If installing on Windows, as there are non-pure Python packages needed for
```

Comment:
how does this look? is this a quote?

Reply:
it is a blockquote. check it out at https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#blockquotes. however, I was worried that this will be broken by the linter requirement on this repo to cap lines at max 80 chars. apparently, it merges successive such quotes into a single one, when rendered.

Reply:
Yes, it's a nice/handy quote, that's fine to use this I would say. (Often using it in Slack)